thesis-de/content-de.tex

```diff
@@ -1892,45 +1892,46 @@ \subsection{SSD Userstory}
 \\textbf{Name der Story:}\\ Anzeige der SSD's, welche einen Hardwaredeffekt haben könnten.
 \\textbf{Beschreibung:}\\ In der heutigen Zeit, werden vermehrt langsame Hard Disk Drive's
 durch schnellere Solid State Disk's ausgetauscht. Gerade im Serverbetrieb, sorgt
-dies für eine schnellere Absicherung der Daten. SSDs haben jedoch, wie auch
-HDDs einen Lebenszyklus, der bei SSD schneller erreicht werden kann. Der
-Administrator soll hier darüber Benachrichtigt werden, sobald von einer SSD
-ein kritischer Schreib- oder Lesezyklus erreicht worden ist, um die SSD, bereits
-vor dem Ausfall austauschen zu können. Ebenfalls soll hier auch eine Statistik
-ausgegeben werden, welche SSD von welchem Hersteller in der Wahrscheinlichkeit
-am häufigsten ausfallen kann.
+dies für einen schnelleren Datenzugriff. SSDs haben jedoch, wie auch HDDs einen
+Lebenszyklus, der bei SSDs schneller erreicht werden kann, als bei 
+handelsüblichen HDDs. Der Administrator soll hier darüber Benachrichtigt werden,
+sobald von einer SSD ein kritischer Schreib- oder Lesezyklus erreicht worden
+ist, um die SSD, bereits vor dem Ausfall austauschen zu können. Ebenfalls soll
+hier auch eine Statistik ausgegeben werden, welche SSD von welchem Hersteller in
+der Wahrscheinlichkeit am häufigsten ausfallen kann.
 \\textbf{Akzeptanzkriterien:}\\
 
 \begin{outline}
-  \1 Die Ausgabe und Visualisierung in einen Diagramm, ob Balken- oder
-  Graphendiagramm, soll jeweils für jede einzelne Instanz angezeigt werden
+  \1 Die Ausgabe und Visualisierung in einem Diagramm, welches vorzugsweise in
```

Comment:
Jetzt beim zweiten mal lesen könnte man doch auch direkt in einem Blaken- oder Graphendiagramm visualisiert schreiben? Jetzt steht das zweimal Diagramm im Text..

Was ist ein Graphendiagramm? Das ist quasi alles -> also Kreis, Balken, Seulen, Plot etc.